src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4423,12 +4424,13 @@ private BoundExpression MakeBadExpressionForObjectCreation(SyntaxNode node, Type
             InitializerExpressionSyntax syntax,
             TypeSymbol type,
             SyntaxNode typeSyntax,
+            bool isNew,
```

Comment:
>isNew [](start = 17, length = 5)

"isForNewInstance"?

Reply:
Still needs to be addressed.

---
In reply to: [554130634](https://github.com/dotnet/roslyn/pull/50200#discussion_r554130634) [](ancestors = 554130634)

Reply:
Thanks.
Note: I merged with latest test from master. I made a small modification to report both the langver and invalid consumption of init-only property.

---
In reply to: [554220332](https://github.com/dotnet/roslyn/pull/50200#discussion_r554220332) [](ancestors = 554220332,554130634)